packages/@lwc/integration-karma/test/component/LightningElement.addEventListener/index.spec.js

```diff
@@ -40,10 +40,7 @@ it('should throw an error if event handler is not a function', () => {
 
     expect(() => {
         document.body.appendChild(elm);
-    }).toThrowConnectedError(
-        Error,
-        /Invariant Violation: Invalid second argument for this\.addEventListener\(\) in \[.*\] for event "click"\. Expected an EventListener but received undefined\./
-    );
+    }).toThrowConnectedError(/Expected an EventListener but received undefined/);
```

Comment:
_notes from rubber duck debugging_
I was wondering why this test changed even though there was no change to base-lightning-element.ts. Digging further I found out(_unrelated to this PR_), this check is being done twice. Once in base-lightning-element.ts:L340 and second in component.ts:L92.

So this test update is to account for those two different assertion message, both asserting the same condition.

Reply:
Yep, exactly. So I chose an error message that fit both cases.